internal/plugins/utils.go

```diff
@@ -19,6 +19,71 @@
 
 package plugins
 
+import (
+	"errors"
+	"fmt"
+	"maps"
+	"slices"
+
+	"github.com/sapcc/go-api-declarations/liquid"
+	"github.com/sapcc/go-bits/logg"
+)
+
 func p2u64(val uint64) *uint64 {
 	return &val
 }
+
+func SortMapKeys[M map[K]V, K ~string, V any](mapToSort M) []K {
```

Comment:
I forgot to nitpick this before because of the larger remark on this function, but `Sort` implies that this function operates in-place, which it does not:
```suggestion
func SortedMapKeys[M map[K]V, K ~string, V any](mapToSort M) []K {
```

Reply:
I included this change in a separate commit, because I changed all referenced to the function with it.